cellpack/autopack/BaseGrid.py

```diff
@@ -758,12 +758,12 @@ def getPointsInCubeFillBB(self, bb, pt, radius, addSP=True, info=False):
         return ptIndices
 
     def test_points_in_bb(self, bb, pt):
-        # given a bounding box, does the point is contains in it
+        # given a bounding box bb, is point pt inside it
```

Comment:
Something is weird with this diff because these changes are already in `main`

Reply:
Oh, I guess it's because we squash merge and this is a branch from your branch

Reply:
Can you rebase on `main` so we can see just the newest changes?

Reply:
Ok, I have a couple formatting changes to go through, and then I'll rebase it on main